src/core/score_map.h

```diff
@@ -79,11 +79,11 @@ class ScoreMap : public DenseSet {
 
   // Returns true if field was added
   // otherwise updates its value and returns false.
-  bool AddOrUpdate(std::string_view field, double value);
+  std::pair<void*, bool> AddOrUpdate(std::string_view field, double value);
```

Comment:
1. You forgot to update the comment
2. Its strange you now have a nice external interface for both sorted set types, but return void* for the element

Reply:
As usual, you are welcomed to fix it up later :)